modules/core/src/main/scala/sangria/validation/QueryValidator.scala

```diff
@@ -148,7 +156,8 @@ class ValidationContext(
     val schema: Schema[_, _],
     val doc: ast.Document,
     val sourceMapper: Option[SourceMapper],
-    val typeInfo: TypeInfo) {
+    val typeInfo: TypeInfo,
+    errorsLimit: Option[Int] = None) {
```

Comment:
Making errorsLimit immutable as `ValidationContext` is a call.
```suggestion
    val errorsLimit: Option[Int] = None) {
```

Reply:
omitting `val` will make it private to the class since it's not a case class, won't it? 🤔 Do we want to make it public?

Reply:
oh yes sorry. Keep it like this.